src/js/constants/ServiceSchema.js

```diff
@@ -45,7 +45,7 @@ let SERVICE_SCHEMA = {
           type: 'number',
           default: 1,
           getter: function (service) {
-            return (service.getInstancesCount() || 0);
+            return (service.getInstancesCount() || 0) + '';
```

Comment:
No! It only affects the style of the input, it will show the stepper controls I believe.